Logic/LoginService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using DAL;
+using Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logic
+{
+    public class LoginService
+    {
+        LoginDAO loginDAO;
+        EncryptionService encryptionService;
```

Comment:
Please put private before the variables if they are not needed outside of this class

Reply:
Fixed!